roles/amq_streams_broker/tasks/main.yml

```diff
@@ -11,6 +11,16 @@
     mode: 0755
   changed_when: false   # TODO: find a better way to deal with that
 
+- name: "Ensure Broker data dir exists and belongs to the appropriate user - if user is defined."
+  ansible.builtin.file:
+    path: "{{ amq_streams_broker_data_dir }}"
+    state: directory
+    owner: "{{ amq_streams_broker_user | default(omit) }}"
+    group: "{{ amq_streams_broker_group | default(omit) }}"
+    mode: 0755
```

Comment:
I'd find 0750 more appropriate, but since it is used as 0755 throughout the roles we can discuss in a different issue